#! /bin/sh /usr/share/dpatch/dpatch-run
## 84-mpc-honour-soversion.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Honour the `version' keyword in MPC files.
## DP: See the following ChangeLog entries:
## DP:  "Mon Nov 29 07:04:58 2004  Chad Elliott  <elliott_c@ociweb.com>"
## DP:  "Tue Nov 23 07:39:23 2004  Chad Elliott  <elliott_c@ociweb.com>"
## DP: This change require setting versioned_so to 1.

@DPATCH@
--- ACE_wrappers.orig/bin/MakeProjectCreator/templates/gnu.mpd	2004-12-02 10:46:25.000000000 +0100
+++ ACE_wrappers/bin/MakeProjectCreator/templates/gnu.mpd	2004-12-02 10:53:44.000000000 +0100
@@ -118,6 +118,12 @@
 tao_dont_use_idl_make_rule = 1
 include $(TAO_ROOT)/rules.tao.GNU
 <%endif%>
+<%if(version)%>
+
+ifeq ($(versioned_so),1)
+SOVERSION = .<%version%>
+endif
+<%endif%>
 
 # To build multiple targets in the same directory on AIX, it works
 # best to have a template directory per project.
